src/Playwright/Playwright.csproj

```diff
@@ -27,6 +27,7 @@
   <Import Project="../Common/SignAssembly.props" />
   <Import Project="build/Microsoft.Playwright.targets" />
   <ItemGroup>
+    <PackageReference Include="lvk.naturally" Version="1.0.2" />
```

Comment:
I'm not sure I want to introduce a new dependency for this, but this let's us target the "latest" nuget package of the current version, due to our semver strategy.

Reply:
We no longer need it, but in general, I would definitely not introduce a dependency for something that can be implemented in three lines, it only adds a security risk.